opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -120,16 +120,17 @@ def __init__(
         self._instrument_temporality = instrument_temporality
         self._instrument_is_monotonic = instrument_is_monotonic
 
-        if self._instrument_temporality is AggregationTemporality.DELTA:
-            self._value = 0
-        else:
-            self._value = None
+        self._current_value = None
```

Comment:
It looks like `self._value` was renamed `self._current_value`, is that right? That generated a lot of changed lines which makes it difficult to review this change. Would you consider changing it back and addressing in a separate PR if it's needed.

Reply:
Sorry for all the renaming of variables. It is actually easier to understand the code now that we have consistent variable naming. I added comments in this PR to let reviewers know which changes are just variable renaming so that they don't have to spend much attention into these changes.